users/tests/test_user_login_logout.py

```diff
@@ -0,0 +1,36 @@
+from rest_framework.test import APITestCase, APIRequestFactory
+from rest_framework import status
+from users.models import User
+from users.tests.factories import UserFactory
+import faker
+
+class LoginTests(APITestCase):
+    def setUp(self):
+        self.user = UserFactory()
+        self.password = faker.Faker().password()
+        self.user.set_password(self.password)
+        self.user.save()
+
+    def test_sign_in_success(self):
+        data=({'email': self.user.email, 'password': self.password})
```

Comment:
Are you using [black](https://github.com/psf/black) or other code formatter?

Reply:
In #4 😛 will fix later.